basalt_runner/src/main.cpp

```diff
@@ -92,34 +92,36 @@ basalt::OpticalFlowBase::Ptr opt_flow_ptr;
 basalt::VioEstimatorBase::Ptr vio;
 
 void load_calibration(std::string calib_path) {
+  std::cout << "Loading standard calibration" << std::endl;
   using json = nlohmann::json;
   std::ifstream ifs(calib_path);
   json config = json::parse(ifs);
   auto cameras = config["cameras"].get<json>();
   for (auto it = cameras.begin(); it != cameras.end(); ++it) {
     json &camera = *it;
-    if (camera.find("imuToCamera") != camera.end()) {
+    if (camera.find("cameraToImu") != camera.end()) {
       Eigen::Matrix4d imuToCamera;
       for (int r = 0; r < 4; r++)
         for (int c = 0; c < 4; c++)
-          imuToCamera(r, c) = camera["imuToCamera"][r][c].get<double>();
+          imuToCamera(r, c) = camera["cameraToImu"][r][c].get<double>();
```

Comment:
The variable names seem wrong/confusing here. `imuToCamera` should be `cameraToImu`? And `T_i_c` means the same thing?